### PR TITLE
Feature/create collection form

### PR DESCRIPTION
**Pull request by Jipperism**

### CODE REVIEW

File: app/collections/create/page.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { CollectionForm } from "@/components/collections/collection-form";
+
+const CreateCollectionPage = async () => {
```
Comment (0xRowdy) [inline app/collections/create/page.tsx:3]: I don't think this needs to be async if you aren't fetching data.

```suggestion
const CreateCollectionPage = () => {
```

File: collections/getCollectionById.ts [hunk 2]
```diff
@@ -2,32 +2,32 @@ import "server-only";
 
 import { graphql, readFragment } from "@/lib/graphql";
 
-import { CollectionFragment } from "./collection.fragment";
+import { HyperboardFragment } from "./hyperboardFragment";
```
Comment (0xRowdy) [inline collections/getCollectionById.ts:5]: alias for import?

File: components/collections/buttons.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog";
+import React from "react";
+import { useDeleteCollection } from "@/collections/hooks";
+
+export const CreateCollectionButton = () => {
+  return (
+    <Button
+      className="hover:text-white rounded-sm bg-white text-black border border-gray-300"
```
Comment (0xRowdy) [inline components/collections/buttons.tsx:22]: I noticed @thebeyondr was updating from grey to slate in his PR. Is that a site-wide change?

**Review by 0xRowdy — APPROVED** [review]
lgtm

File: app/collections/create/page.tsx (hunk 1 above)
Comment (Jipperism) [reply app/collections/create/page.tsx]: fixed

File: collections/getCollectionById.ts (hunk 2 above)
Comment (Jipperism) [reply collections/getCollectionById.ts]: fixed

File: components/collections/buttons.tsx (hunk 3 above)
Comment (Jipperism) [reply components/collections/buttons.tsx:22]: I wasn't aware and not sure if it's a site-wide change. If it is, I don't think this should be applied to each button separately. Will bring it up in the dev checkin in an hour.

**Comment by Jipperism** [comment]
Closes https://github.com/hypercerts-org/hypercerts-api/issues/163
Closes #235 
Closes #237
Closes #236 
Closes #245 
Closes #239

**Pull request merged**